File: 217/217.cpp
/*!***********************
 * @brief:  leetcode-217. 存在重复元素
 * @date:   2019/5/17
 * @envir:  gcc 7.4.0
 * @author: luhao
 * ***********************/

class Solution {
public:
    bool containsDuplicate(vector<int>& nums) {
        sort(nums.begin(), nums.end());
        for (int i=0; i<nums.size()-1; i++){
            if (nums[i] == nums[i+1])
               return 1;
        }
        return 0;
    }
};

File: 217/217.py
# @brief:  leetcode-217. 存在重复元素
# @envir:  python 2.7
# @date:   2019/5/17
# @author: luhao

class Solution(object):
    def containsDuplicate(self, nums):
        """
        :type nums: List[int]
        :rtype: bool
        """
        return len(set(nums))!=len(nums)
    
